app/models/qernel/area.rb

```diff
@@ -32,6 +32,14 @@ def inspect
       "<Area #{area_code}>"
     end
 
+    def initializer_inputs
+      begin
+        Atlas::Dataset::Derived.find(area_code).initializer_inputs
```

Comment:
@antw I know that this is not the way to do this, but I didn't quiet understand how else to approach this problem. Could you maybe show me what a nicer way would be? 😄

Reply:
On `master`, the data should already be available on the `Qernel::Area` with the same name as in Atlas (`init`).

```ruby
gql = Scenario.default(area_code: 'noorderplantsoen').gql

gql.future.graph.area.init
# => { :households_useful_demand_cooking_per_person=>717610.2,
#      :households_cooker_halogen_electricity_share=>33.6,
#      ...
```

That's because Atlas attributes are automatically copied across to the `Qernel::Area`:

https://github.com/quintel/etengine/blob/9d21f2fcee31818785092129eff782c1dca4ff16/app/models/qernel/area.rb#L7-L15

I think you can achieve the same thing for your new method like so:

```ruby
ATTRIBUTES_USED = 
  [Atlas::Dataset::Full, Atlas::Dataset::Derived]. 
    .map { |klass| klass.attribute_set.map(&:name) }. 
    flatten +
    [:initializer_inputs] -
    [:id, :parent_id] 
```